applications/CompressiblePotentialFlowApplication/tests/test_CompressiblePotentialFlowApplication.py

```diff
@@ -5,6 +5,8 @@
 import run_cpp_unit_tests
 
 ##### SMALL TESTS #####
+from potential_flow_test_factory import Naca0012SmallTest
+from potential_flow_test_factory import SmallLiftJumpTest
```

Comment:
If you define the SmallLiftJumpTests inside the PotentialFlowTests class, these lines can be removed
```suggestion

```